src/compiler/diagnosticMessages.json

```diff
@@ -1279,7 +1279,7 @@
         "category": "Error",
         "code": 2409
     },
-    "All symbols within a 'with' block will be resolved to 'any'.": {
+    "Unsupported 'with' statement, all symbols within a 'with' block will be resolved to 'any'.": {

```

Comment:
I would word the error as "The 'with' statement is not supported. All symbols in a 'with' block will have type 'any'."